sc-tools/sc-server/src/sc-server-impl/sc-memory-json/sc-memory-json-action/sc_memory_json_action.hpp

```diff
@@ -6,15 +6,15 @@
 
 #pragma once
 
-#include <sc-memory/sc_memory.hpp>
-
 #include "sc-server-impl/sc-memory-json/sc_memory_json_payload.hpp"
 
+class ScAgentContext;
```

Comment:
why don't you include file with sc agent context?

Reply:
It is hpp file